parent/pom.xml

```diff
@@ -211,6 +212,13 @@
                 <version>3.0.20</version>
             </dependency>
 
+            <!-- Doclet for inclusion of code snippets -->
+            <dependency>
+                <groupId>org.apidesign.javadoc</groupId>
+                <artifactId>codesnippet-doclet</artifactId>
+                <version>${org.apidesign.javadoc.codesnippit.version}</version>
+            </dependency>
```

Comment:
This is not required here, as the version needs to be managed at the javadoc plugin dependencies.